waspc/data/Generator/templates/react-app/vite.config.ts

```diff
@@ -30,6 +30,7 @@ const defaultViteConfig = {
     outDir: "build",
   },
   resolve: {
+    conditions: ["client-runtime"],
```

Comment:
This is where we set the framework runtime's condition string, thus gaining access to the SDK's private API.